sklearn/discriminant_analysis.py

```diff
@@ -25,47 +25,63 @@
 from .utils.extmath import softmax
 from .preprocessing import StandardScaler
 
-
 __all__ = ['LinearDiscriminantAnalysis', 'QuadraticDiscriminantAnalysis']
 
 
-def _cov(X, shrinkage=None):
-    """Estimate covariance matrix (using optional shrinkage).
-
+def _cov(X, shrinkage=None, covariance_estimator=None, standardize=False):
+    """Estimate covariance matrix (using covariance_estimator).
     Parameters
     ----------
     X : array-like, shape (n_samples, n_features)
         Input data.
-
+    covariance_estimator : BaseEstimator
+        A covariance estimator, possible values
+            - None: shrinkage parameter drives the estimate
+            - BaseEstimator with a fit method and
+            a .covariance_ attribute like the estimators
```

Comment:
I think you need an extra indentation here (checking the rendered version is a good idea (https://scikit-learn.org/dev/developers/contributing.html#generated-documentation-on-circleci))